docs/src/components/LeftSidebar/LeftSidebar.astro

```diff
@@ -32,6 +32,15 @@ const sidebarSections = SIDEBAR[langCode].reduce((col, item) => {
         </div>
       </li>
     ))}
+    <li>
+      <div class="nav-group">
+        <h2 class="nav-group-title">Sponsors</h2>
+        <ul class="sponsors">
+          <li><a href="https://www.netlify.com/"><img width="74" height="20" src="/assets/netlify.svg" alt="Netlify" /></a></li>
+        </ul>
+        <a class="sponsors-signup" href="https://opencollective.com/astrodotbuild" target="_blank">Add your logo</a>
```

Comment:
Become a Sponsor > Add your logo.

Reply:
alternatively I'd be fine to not ship this button for now. I couldn't find a way to make it look nice, given that it has to live in a sidebar with other clickable links.